react-training/src/app/screens/Game/index.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+import Board from './Board';
+import styles from './styles.scss';
+
+class Game extends Component {
+    state = {
+      history: [
+        {
+          squares: Array(9).fill(null)
+        }
+      ],
+      stepNumber: 0,
+      xIsNext: true
+    };
+  
+    calculateWinner = (squares) => {
+      const lines = [
```

Comment:
To avoid to declare this const in every calculation, move this const to a separated file, in `src/constants/lines.js`, importe it here and use here.